runtime/chainx/src/lib.rs

```diff
@@ -1076,7 +1076,7 @@ construct_runtime!(
 
         // ChainX basics.
         XSystem: xpallet_system::{Pallet, Call, Storage, Event<T>, Config} = 24,
-        XAssetsRegistrar: xpallet_assets_registrar::{Pallet, Call, Storage, Event, Config} = 25,
+        XAssetsRegistrar: xpallet_assets_registrar::{Pallet, Call, Storage, Event<T>, Config} = 25,
```

Comment:
`type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;` has defined in pallet::Config, it's unnecessory